etc/kconfigs.yaml

```diff
@@ -276,6 +276,7 @@ USB4: v5.6-rc1
 USB4_KUNIT_TEST: v5.9-rc1
 USER_EVENTS: v5.18-rc1
 USERFAULTFD: v4.3-rc1
+V4L_TEST_DRIVERS: v5.8-rc1
```

Comment:
looks no need to have this, as it builds a test driver not something that runs test.